batch.py

```diff
@@ -706,6 +706,7 @@ def crm_pipeline(
         )
         _plink_genes |= set(extract_genes(genes_of_interest, expression_tsv_path))
     plink_genes = list(sorted(_plink_genes))
+    logging.info(f'Done selecting genes, total number: {len(plink_genes)}')
```

Comment:
Can you add a log statement right before _and_ after each `extract_genes` and `glob`?

Reply:
let me know if i missed any!